src/App.jsx

```diff
@@ -14,57 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [wasSelected, setWasSelected] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  function clearClass() {
+    const allGoods = document.querySelectorAll('[data-cy="Good"]');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+    allGoods.forEach(good => {
+      good.classList.remove('has-background-success-light');
+    });
+  }
```

Comment:
The `clearClass` function directly manipulates the DOM using `document.querySelectorAll`. This approach can lead to issues in React, as it bypasses React's virtual DOM. Consider using React refs or managing the class state within React to ensure consistent behavior across renders.